components/blocks/refCard.module.css

```diff
@@ -1,5 +1,9 @@
 .Container {
-  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-70;
+  @apply rounded-md border border-gray-30 overflow-hidden text-base cursor-pointer transition-all duration-75 flex flex-col hover:scale-105 hover:border-b-gray-30 hover:opacity-90;
+}
+
+.Container:hover {
+  transform: scale(1.05);
```

Comment:
When hovering over reference cards, the text becomes a bit blurry:

![Screen Shot 2022-09-06 at 1 11 06 PM](https://user-images.githubusercontent.com/34423371/188684426-4de24a8a-c17d-422d-a5a4-04a7aac46924.png)

I'm not sure if it's the `scale` or the `opacity` (or both) causing this, but I wonder if that's desirable, especially on API reference cards which feature quite a bit of text (and also code snippets)

Reply:
Oh good catch! I missed the blurring without my glasses on 😅 We definitely do not want any text blurring, especially on API reference cards

Reply:
`hover:opacity-90` was causing the blurring. It is removed when `hover:opacity-90` is removed.